### DISCUSSION TITLE
ATSC 3.0 tuners

### DISCUSSION THREAD

**Discussion by bds3151** [djp952/pvr.hdhomerundvr#124] [General]
Does this add-on support the new HDhomerun ATSC 3.0 tuners? If not, is there a plan to support these at a future date?

**Comment by djp952** [comment]
It should, but I don't have an ATSC3 tuner to test? Reports to date have been that the addon works as expected, but Kodi may not due to ATSC 3 audio/video decoding concerns. I think if you are running Kodi 20 "Nexus" it's expected to be better, if not "good"?

I apologize for a semi-terse response on this, but a couple/few months ago I opted to move this project into "required maintenance only" mode; it's no longer being actively developed. Things will start to fall off and start to fail and my ability to deal with those things will lessen over time. I am still a Kodi Windows user, so that baseline will live the longest since I need to use it too, but as of right now, I am (perhaps sadly) inclined to just let this project die.

To my knowledge, there are no issues with the addon and ATSC 3.0 devices, and I am open to being wrong about this, but HDHomeRun devices supply a basic MPEG-TS stream to any client(s), and it's up to the client(s) to be able to decode that MPEG-TS stream properly. When it comes to data, the addon does little to nothing with it; it just passes along the data from the tuner(s) or the RECORD engine(s).

Welcome to my "this has become unsustainable as a hobby project" world :)

**Reply by bds3151** [reply]
Ok - thank for your effort to date on this great kodi addon. I'm a windows/kodi user also and was thinking about getting an ATSC 3.0 tuner and just wanted to investigate to see if it would work.

**Reply by noticons** [reply]
> Welcome to my "this has become unsustainable as a hobby project" world :) 

Is there anything we can do to help support? I think this is a great project and would love to support it, if possible. It works perfectly and it integrates so seamlessly with HDHomeRun.

**Reply by djp952** [reply]
Sorry I didn't see this and marked this as closed on you, re-opened!

I mean, the project seems like it's running fine just being on life support? I just updated it for Omega today (and tweaked Nexus to auto-update to Omega). To my knowledge, there is still nothing wrong with it other than it's been ignored for a year?

The main time dependency is with dealing with all the various baselines (Windows, Android, OS-X, Linux) and trying to keep all the dependencies up to date. It takes many, many hours just to update one and then it needs to be tested over and over on as many platforms as possible. I've pared down my setup to basically just Windows again, but I do still have Android to test :) At some point something will go wrong!

I also never supported iPhone, and now I see they also have webOS for Omega. These binary Kodi addons are very difficult to port, and you really do need to have access to the hardware. ATSC 3.0 and IPv6 from the HDHomeRun side are also hardware dependent, I don't have that hardware so I really can't test it.

I still use this every day, for hours and hours in fact. If it breaks I'll still fix it. I'm just content to let as many sleeping dogs lie as I can and not spend hours upon hours every week looking for updates to make and fighting with them :)

I do wish somebody more close with Team Kodi forked this and made some version of it "official", but I don't think that will ever happen.

Thanks for the support and of course for continuing to use it! When it finally breaks, let me know!

**Reply by noticons** [reply]
Thank you! Everything is working excellent, as I said, I was really just thinking about monetary support. I wouldn't mind donating some money out of appreciation because there isn't anything else that even comes close! The only thing that I don't quite have lined up with pvr.hdhomerundvr is that the build ARMhf is, as I understand it, for 32bit. I'm running libreelec ("just enough OS for Kodi") on a Pi4, which is 64 bit ARM. But it still works well.

**Reply by noticons** [reply]
Oh gosh. I just realized what a mistake I made. The release client is the aarch64 for raspberry pi. I don't know what I did to make me think I used the HF variety. I just upgraded my kodi-boxes to libreelec 12 and did the install on the new HDhomerundvr and realized which type I was using. Thank you so much for an incredible addon.

**Comment by djp952** [comment]
I appreciate the kudos, and I do think that an ATSC 3.0 tuner will work, but I am not in a position to control how well that tuner will work in Kodi :(

**Reply by bds3151** [reply]
I bought the new HD atsc3 tuner and it displays a picture but no sound on ATSC 3 channels in Kodi 20. It works of course on the ATSC1 channels. Not asking you to do anything but I thought I'd pass on the info. I think it because the new audio format (AC4) is not yet supported by Kodi.
